composer/loss/loss.py

```diff
@@ -157,6 +158,123 @@ def soft_cross_entropy(input: Tensor,
         raise ValueError(f'Unrecognized target type {target_type}')
 
 
+class DiceLoss(_Loss):
```

Comment:
does this need to be a class? The other losses here are functions.

Reply:
I was torn on this, so happy to hear your opinion. This could be turned into a function, but I would need to use `functools.partial` in DeepLabv3's `model.py` since the loss needs new default arguments. I thought it would be cleaner to make a class, then to have a function and use `functools.partial`. But I don't have a strong opinion here